### PR TITLE
Update Ruby and Rails

### PR DESCRIPTION
**Pull request by frankyston**
Foi atualizado as versões do Ruby e do Rails

### CODE REVIEW

File: spec/factories/event.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    name        'GURU-SP'
+    description '50th meeting'
+    association :user
+    occurs_at   { 1.month.from_now }
+
+    trait :ruby do
+      after(:create) { |event| event.tag_list.add("ruby"); event.save }
+    end
+  end
+end
```
Comment (lucasrenan) [inline spec/factories/event.rb]: @frankyston minor change here, please: https://stackoverflow.com/a/5813359

**Comment by lucasrenan** [comment]
@frankyston write the descriptions in english, please. thanks for your contribution :)

**Comment by lucasrenan** [comment]
@frankyston please also update the `Dockerfile` and the the env files: https://github.com/call4paperz/call4paperz/tree/master/config/environments

**Comment by lucasrenan** [comment]
I suggest you following this guide: https://guides.rubyonrails.org/upgrading_ruby_on_rails.html

File: spec/factories/event.rb (hunk 1 above)
Comment (frankyston) [reply spec/factories/event.rb]: Done. 2660fd95c3df39691d91e2cfdffe858821ef993c

**Comment by frankyston** [comment]
> I suggest you follow this guide: https://guides.rubyonrails.org/upgrading_ruby_on_rails.html

Done 610132a3b3abb5b30df4f4d5edeaaee44a1261ac

File: config/initializers/assets.rb [hunk 2]
```diff
@@ -11,4 +11,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
-# Rails.application.config.assets.precompile += %w( 404.css )
+# Rails.application.config.assets.precompile += %w( admin.js admin.css )
```
Comment (lucasrenan) [inline config/initializers/assets.rb]: ?

File: spec/factories/event.rb (hunk 1 above)
Comment (lucasrenan) [reply spec/factories/event.rb]: I still the "no newline at end of file" issue

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.5.1-slim
+FROM ruby:2.6.2-slim
 
 RUN apt-get update && \
     apt-get install -y build-essential nodejs libpq-dev postgresql-client imagemagick
```
Comment (lucasrenan) [inline Dockerfile]: @frankyston I'll kindly ask you to update ruby and rails in two separate PRs. 

I just notice the package `postresql-client` have changed on `debian:stretch-slim` version used by `ruby:2.6.2-slim` image. without the proper fix, current call4paperz docker image does not build.

a subset of the stack trace when trying to build the image:

```
Errors were encountered while processing:
 postgresql-client-9.6
 postgresql-client
E: Sub-process /usr/bin/dpkg returned an error code (1)
ERROR: Service 'web' failed to build: The command '/bin/sh -c apt-get update &&     apt-get install -y build-essential nodejs libpq-dev postgresql-client imagemagick' returned a non-zero code: 100
```

Comment (frankyston) [reply Dockerfile]: @lucasrenan Ok, I'm going to do this in just the rails and ruby in another PR.

**Pull request closed without merging**